config/routes.rb

```diff
@@ -67,6 +65,10 @@
   end
   delete "users/me/relationships/favorite-tools", to: "favorite_tools#destroy"
 
+  scope "users/me" do
```

Comment:
I forgot to mention that this should leverage the pre-existing `uesrs/:user_id` logic for the route. that logic will resolve `user_id=me` to be the current user and in the future support admin management functionality for managing other users if necessary.

This change should also apply to the `relationships/favorite-tools` endpoint as well